### ISSUE TITLE
PushSecret/Kubernetes: allow cloning of ConfigMaps across clusters

### ISSUE THREAD

**Issue by ron1** [external-secrets/external-secrets#2841]
As a user i want to be able to clone a configmap from one cluster to another using PushSecret. At the moment we can only push individual keys from a secret, not individual keys from a configmap nor a whole configmap. Since ConfigMaps are first-class Kubernetes resources that provide a subset of the functionality offered by Secrets, consider expanding the scope of the Kubernetes provider PushSecret to also support ConfigMaps.

Prerequisites: https://github.com/external-secrets/external-secrets/issues/2836

AC:

 users are able to clone a configmap across clusters using PushSecret
 end-to-end tests cover this scenario

**Comment by pre** [comment]
Cloning of labels of the ConfigMap should also be supported. 

See https://github.com/external-secrets/external-secrets/issues/2836#issuecomment-1793377172

**Comment by evrardj-roche** [comment]
This issue has been marked as Stale and will be closed soon if there is no activity. Please update the issue or engage with it to keep it open. It is fine to re-open this issue in the future if you want to work on this.

**Comment by ron1** [comment]
I am still interested in this feature.

**Comment by Skarlso** [comment]
Technically, this can be achieved with Custom Targets now.... but the feature is in alpha atm and undergoing some patch fixes.

**Comment by ron1** [comment]
Yes, feature https://github.com/external-secrets/external-secrets/blob/413d1f9c07d4318dc419dc33e0d08d6616f03b02/docs/guides/targeting-custom-resources.md definitely satisfies this requirement, thanks. Does a feature request like this typically typically get closed when the feature reaches alpha?

**Comment by evrardj-roche** [comment]
Hello. Thank you for taking the time to open this issue and contribute to the project.
We truly appreciate community involvement and the effort that goes into reporting issues and suggesting improvements.

We're closing this (Stale) issue as part of a maintenance effort to keep our issue tracker focused and actionable.
Issues that have been inactive for an extended period can create clutter, make it harder to prioritize current work, and distract maintainers from issues that have active contributors ready to work on them.

This doesn't mean your issue isn't valid or important.
If you or anyone else is still affected by this issue and willing to contribute toward a solution (whether through discussion, testing, or code),
please feel free to comment here, reopen the issue, and mention you want to be assigned on this issue.

Alternatively, you can directly submit a PR with `Closes <PR NUMBER>` in its content.

We're happy to reopen or revisit this if there's renewed interest and someone willing to champion it forward.

Thanks again for being part of our community!
